README.md

```diff
@@ -1,133 +1,64 @@
 
-As usual please start by forking this repo.
+<img src='img/animation.gif'>
 
-We are going to write a small Twitter clone that will allow the users to post messages to a public stream.
+## Chitter
 
-Features:
--------
+Like Twitter but way better
 
-```
-STRAIGHT UP
+Using:
+- Ruby
+- Sinatra
+- PostgreSQL database
+- DataMapper ORM
+- RSpec/Capybara for testing
+- Bootstrap CSS
+- Heroku + Heroku Postgres
 
-As a Maker
-So that I can let people know what I am doing  
-I want to post a message (peep) to chitter
+Check it out here: <https://hivemind-chitter.herokuapp.com/>
 
-As a maker
-So that I can see what others are saying  
-I want to see all peeps in reverse chronological order
+## Installation
 
-As a Maker
-So that I can better appreciate the context of a peep
-I want to see the time at which it was made
+1. Clone the repo
+2. `bundle install`
+3. Set up your databases and user e.g. by running:  
+   `psql -f db/migrations/set_up_databases.sql`
+4. Copy `.env-example` to `.env`.(`.env` is included in `.gitignore` in case you want to keep your database credentials secret)
+5. Start the server with `rackup`
+6. Go to `localhost:9292` in your browser
 
-As a Maker
-So that I can post messages on Chitter as me
-I want to sign up for Chitter
+## Tests
 
-HARDER
+1. Run the tests with `rspec` in the project root
 
-As a Maker
-So that only I can post messages on Chitter as me
-I want to log in to Chitter
+## What I did
 
-As a Maker
-So that I can avoid others posting messages on Chitter as me
-I want to log out of Chitter
+I implemented all the user stories except the last one. I didn't look at the bonus task about replying to peeps. I did a little bit of styling with Bootstrap.
 
-ADVANCED
+I used DataMapper for an ORM (seems *much* easier than dealing with the database directly!)
 
-As a Maker
-So that I can stay constantly tapped in to the shouty box of Chitter
-I want to receive an email if I am tagged in a Peep
-```
+It took me a couple of goes to set the database up correctly, but the Travis build is passing.
 
-Technical Approach:
------
+I deployed to Heroku using the Heroku CLI. Heroku Postgres was pretty easy to configure via DataMapper.
 
-This week you integrated a database into Bookmark Manager using the `PG` gem and `SQL` queries. You can continue to use this approach when building Chitter Challenge.
+Databases are uniquely named (i.e. called "hivemind_chitter" instead of just "chitter") so they won't conflict with anyone else's that I download and install.
 
-If you'd like more technical challenge this weekend, try using an [Object Relational Mapper](https://en.wikipedia.org/wiki/Object-relational_mapping) as the database interface.
+TDD-ed everything, but almost all the model logic was handled by DataMapper, so there wasn't much in the way of unit tests, which felt... weird... Test coverage is ~98%: everything except the lines setting up the database in the development and 'production' environment.
```

Comment:
This is really helpful, however for final review purposes, maybe you could look at including your process? It would show how you're improving your approach to solving bugs and building the framework of a web app.

Reply:
fair point... in my defence it was late and my brain was frazzled.